### DISCUSSION TITLE
SHA256 mismatch for cask download

### DISCUSSION THREAD

**Discussion by Danrancan** [Homebrew/discussions#606] [Casks]
I'm getting a mismatch error with my checksums when trying to download lingon-x. Could this be due to a mitm attack, or is it something more obvious?
```
$brew install lingon-x
==> Downloading https://www.peterborgapps.com/downloads/LingonX8.zip
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: c1b61423ac76dc5e76934db0d00697a344eacf34b526d60cc183d712a7d51010
  Actual: 04855f755dd7d15ab13fb9dc224495e03cad7276adbd2737fa4339b148fdf5af
```

**Comment by gromgit** [comment]
Upstream (silently) upgraded their ZIP to v8.0.5. https://github.com/Homebrew/homebrew-cask/pull/98645 has just been merged, so `brew update` and try again.

**Reply by Danrancan** [reply]
Brew update has no affect and results in same error. Any suggestions?

**Reply by gromgit** [reply]
> Brew update has no affect

If you meant that literally, your Homebrew setup is b0rked. What's the output of the following:
```
brew info homebrew/cask/lingon-x
brew config
brew doctor
```

**Reply by tomuGo** [reply]
@Danrancan same

**Reply by tomuGo** [reply]
> > Brew update has no affect
> 
> If you meant that literally, your Homebrew setup is b0rked. What's the output of the following:
> 
> ```
> brew info homebrew/cask/lingon-x
> brew config
> brew doctor
> ```

i did those,but brew install pyqt,SGA256 mismatch.is mac m1 did not work?

**Reply by SMillerDev** [reply]
> i did those,but brew install pyqt,SGA256 mismatch.is mac m1 did not work?

What is the output? Without it we can't possibly help you
